package.json

```
@@ -11,14 +11,15 @@
     "dev": "turbo run dev --filter=@channel.io/bezier-react",
     "lint": "turbo run lint && syncpack lint",
     "format": "prettier . --write",
-    "typecheck": "turbo run typecheck",
+    "typecheck": "turbo run typecheck && yarn knip --include exports,types,nsExports,nsTypes",
```

knip.json 으로 config를 설정했는데 --include로 또 옵션을 지정할 필요는 없지 않을까요? --include 옵션이 knip.json 설정을 오버라이딩하는 것 같아서 지우고 knip.json 설정을 따라가면 어떨까요? 

```suggestion
"typecheck": "turbo run typecheck && yarn knip",
```

---

빼도 문제 없긴 해요...!!
다만 저 커맨드가 `typecheck`인데, unused files까지 검사하는 로직이 들어있는 게 이상한 것 같아서 뻈던 것 같아요 
다시 생각해보니까 `knip` 자체가 타입 체킹보다는 dead code 검사하는 애라서 저 커맨드에 &&으로 합쳐져 있는 게 이상한 것 같긴 하네요 🤔

---

엄밀히 따지면 말씀하시는게 맞긴 합니다만, 나름 편한 측면도 있더라고요. 컴포넌트 만들었는데 index.ts 에서 export 안한다거나 하는 실수를 방지할 수 있어서 typecheck 에 포함시켜도 저는 괜찮다고 생각합니다!